app/views/games/_divisions.html.haml

```diff
@@ -8,7 +8,7 @@
     .tab-content
       - divisions.each do |d|
         .tab-pane{ :class => ('active' if active_division.eql? d), :id => "division-#{d.id}-tab" }
-          = render partial: "games/team_list", locals: { teams: d.ordered_teams(only_top_five) }
+          = render partial: "games/team_list", locals: { teams: d.ordered_teams(only_top_five), display_all: !only_top_five}
```

Comment:
This looks like it requires a page reload every time, it should be possible to just load all the results on page load and then hide and show with JavaScript.